drivers/serial/uart_ra.c

```diff
@@ -370,8 +368,8 @@ static const struct uart_driver_api uart_ra_driver_api = {
 		.pcfg = PINCTRL_DT_DEV_CONFIG_GET(DT_INST_PARENT(n)),                              \
 	}
 
-#define UART_RCAR_INIT(n)							\
-	UART_RCAR_INIT_CFG(n);                                                  \
+#define UART_RA_INIT(n)							\
+	UART_RA_INIT_CFG(n);                                                  \
```

Comment:
non blocking but fyi there's a mix of tabs and spaces for padding the `\` here, worth fixing if you end up touching this file again

Reply:
Thank you for your review.

I ran clang-format again, so it should be fixed.

Reply:
thanks, actually would you mind checking the other files as well? spotted a couple more indentation that looks off, may fix that as well while we are at it

Reply:
There's no strong reason to decline it, so let's do it that way.

Reply:
There weren't any problems with tabs and spaces.
However, there are problems with the indentation of the function, so I added a commit to format it.

Reply:
thanks!